Unreal/Plugins/AirSim/Source/Vehicles/CustomCar/CustomCarPawn.cpp

```diff
@@ -84,6 +86,11 @@ void ACustomCarPawn::setupVehicleMovementComponent()
 void ACustomCarPawn::setVehicleModelInput(VehicleInput vehicle_input)
 {
     vehicle_model_.setVehicleInput(vehicle_input);
+    float steering = vehicle_input.steering_angle; 
+    if (steering < -8.48f) steering = -8.48f;
+    else if (steering > 8.48f) steering = 8.48f;
```

Comment:
there is a "clamp" function that can be used for this.

Reply:
Added this